build.gradle

```diff
@@ -511,6 +511,7 @@ if (jdkVersion == '8') {
 
     // exclude these Hive2-specific tests from iceberg-mr
     test {
+      exclude '**/TestHiveSchemaUtil.class'
```

Comment:
Why is this specific to Hive 2?

Reply:
We had a specific test for Hive3 (TestHive3SchemaUtil.java). Removed now, since it is Timestamp related